From 174ec6666308bcf1e8083ba37bde653941dd8ebd Mon Sep 17 00:00:00 2001
From: Amin Bandali <bandali@gnu.org>
Date: Sat, 25 Feb 2023 14:57:23 -0500
Subject: [PATCH] * debian/jami.lintian-overrides: Drop vestigial file.

This was previously added likely because the jami package used to ship
a libringclient.a static library for use by the client, but after an
upstream refactoring merging the Jami libclient sources into
jami-client-qt.git, that is no longer needed.
---
 debian/jami.lintian-overrides | 3 ---
 1 file changed, 3 deletions(-)
 delete mode 100644 debian/jami.lintian-overrides

diff --git a/debian/jami.lintian-overrides b/debian/jami.lintian-overrides
deleted file mode 100644
index 8ab14fe9..00000000
--- a/debian/jami.lintian-overrides
+++ /dev/null
@@ -1,3 +0,0 @@
-# We are shipping a .so, but this isn't a library package
-package-name-doesnt-match-sonames
-no-symbols-control-file
-- 
GitLab

